tests/sdk/go/helm-release-import/step1/nginx/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
This is the same as the chart in the local chart example. Its a lot easier to refer to a local chart with the manual chart install and figured its best to copy it. Let me know if we should consolidate instead.

Reply:
Fine by me